scheduler_plugins/slurm/utils/upload_artifacts.sh

```diff
@@ -19,6 +19,11 @@ function usage {
     exit 1
 }
 
+BUILD_OS=$(uname)
+if [ "${BUILD_OS}" != "Linux" ]; then
+  SED_BAK_EXT="''"
```

Comment:
Afaik the safest approach to reuse the same sed parameters both in Linux and Mac is to use `gsed` instead of `sed` on Mac.
Btw, no problem at all with this approach as far as it's working on our specific case. 
Thanks for proposing it: I did not know about it.